sdk/servicebus/azure-servicebus/tests/async_tests/test_queues_async.py

```diff
@@ -2123,3 +2124,53 @@ async def test_queue_async_send_amqp_annotated_message(self, servicebus_namespac
                     assert recv_sequence_msg == 3
                     assert recv_value_msg == 3
                     assert normal_msg == 4
+
+
+    @pytest.mark.liveTest
+    @pytest.mark.live_test_only
+    @CachedResourceGroupPreparer(name_prefix='servicebustest')
+    @CachedServiceBusNamespacePreparer(name_prefix='servicebustest')
+    @ServiceBusQueuePreparer(name_prefix='servicebustest', dead_lettering_on_message_expiration=True)
+    async def test_message_state_scheduled_async(self, servicebus_namespace_connection_string, servicebus_queue, **kwargs):
```

Comment:
biggest baby nit:
I know the SB tests take a long time to run, and I'm not sure whether separating the message state tests into 2 makes them run longer?
If so, would it be possible to combine these two tests into one?
If not, ignore :)

Reply:
combined testing active and deferred - would like to separate scheduled and deferred for clarity purposes